testutil/integration/network/config.go

```diff
@@ -25,6 +28,7 @@ func DefaultConfig() Config {
 	account, _ := testtx.NewAccAddressAndKey()
 	return Config{
 		chainID:            utils.MainnetChainID + "-1",
+		eip155ChainID:      big.NewInt(9001),
```

Comment:
Why this hardcoded number? The const used at the line above has this docstring:

```
// MainnetChainID defines the Evmos EIP155 chain ID for mainnet
MainnetChainID = "evmos_9001"
```

Isn't this one the eip155 chain id?